package.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Please remove. There is no need to coerce a Ruby project to dress up as a JS project